spec/implementation.rb

```diff
@@ -16,6 +17,33 @@ def configure
 
   private
 
+  def start_remote_server
+    require 'selenium/server'
+
+    @server ||= Selenium::Server.new(remote_server_jar,
+                                     :port       => Selenium::WebDriver::PortProber.above(4444),
+                                     :log        => !!$DEBUG,
+                                     :background => true,
+                                     :timeout    => 60)
+
+    @server.start
+    at_exit { @server.stop }
+  end
+
+  def remote_server_jar
```

Comment:
Can't [Selenium download](https://github.com/SeleniumHQ/selenium/blob/master/rb/lib/selenium/server.rb#L39-L42) it?